### PR TITLE
Add unit tests for com.learnwithted.kidbank.domain.Percent

### PR DESCRIPTION
**Pull request by ThomasPerkins1123**
Hi,

I've analysed your code base and noticed that `com.learnwithted.kidbank.domain.Percent` in the `kid-bank` module is not fully tested.

I've written some tests that cover this class with the help of [Diffblue Cover](https://www.diffblue.com/opensource).

Hopefully, these tests should help you detect any regressions caused by future code changes. If you would find it useful to have additional tests written for this repository, I would be more than happy to look at other particular classes that you consider important.

### CODE REVIEW

File: src/test/java/com/learnwithted/kidbank/domain/TestPercent.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package com.learnwithted.kidbank.domain;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TestPercent {
```
Comment (Braavos6) [inline src/test/java/com/learnwithted/kidbank/domain/TestPercent.java]: Change the name so the class name comes 1st

**Review by EricHetti — CHANGES_REQUESTED** [review]
Since the class is super small.. I would add more test cases to it even though you already have 100% coverage.. Just so it looks complicated :)

File: src/test/java/com/learnwithted/kidbank/domain/PercentTest.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  @Test
+  public void testToString() {
+    Assert.assertEquals("100%",
+            Percent.of(1).over(0).toString());
+    Assert.assertEquals("50%",
+            Percent.of(1).over(2).toString());
+  }
+
+  @Test
+  public void testRoundToInteger() {
```
Comment (petersear) [inline src/test/java/com/learnwithted/kidbank/domain/PercentTest.java:19]: You could add a test that returns `99` (the RHS of the ternary operator on line 29 of roundToInteger())

Comment (ThomasPerkins1123) [reply src/test/java/com/learnwithted/kidbank/domain/PercentTest.java:19]: Thank you I always miss these when they use the single line if statements

**Comment by paulkennethkent** [comment]
I have some tests that I can add to this @ThomasPerkins1123, I will push some of my changes

**Review by paulkennethkent — COMMENTED** [review]
Please add the other tests if suitable and possibly rename the branch.

**Pull request closed without merging**